.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,16 @@
+name: Feature Request
+description: Got an idea for a feature that Gitea doesn't have currently?  Submit your idea here!
+body:
```

Comment:
We should also add some notes here about using the English language, and about checking first that the feature hasn't already been requested.

Reply:
We don't have much space here, but I could add a markdown system similar to the bug report to the feature template.

Reply:
I've gone ahead and added a message to the top of the feature request template.